spec/models/timecard_spec.rb

```diff
@@ -42,7 +34,7 @@
       @event = create(:event)
       @person = create(:person)
       @original_timecard = create(:timecard, event: @event, person: @person,
-                                              intended_start_time: 42.minutes.from_now, 
+                                              intended_start_time: 42.minutes.from_now,
                                               intended_end_time: 8.hours.from_now,
                                               intention: "Available")
       @duplicate_timecard = build(:timecard, event: @event, person: @person,
```

Comment:
and again at https://github.com/ReadyResponder/ReadyResponder/pull/357/commits/a83b14cfcc87e4fe58d921224258b18bec3bafd0#diff-72c3b2cfe4c8abd673db9f4d72d2a2efR45